### ISSUE TITLE
Aligner for headless text fields/areas, similar to Button.Aligner

### ISSUE THREAD

**Issue by b-graves** [formaat-design/reshaped#180]
**Is your feature request related to a problem? Please describe.**
 
Headless text fields/area still have padding, so become misaligned from the text. See example:

<img width="670" alt="Screenshot 2023-10-05 at 14 55 51" src="https://github.com/formaat-design/reshaped/assets/56936483/f076e763-b1a3-45c8-83c2-429bf50e3e44">

**Describe the solution you'd like**
Either no padding on headless options or an Aligner the same was we have for ghost Buttons

**Describe alternatives you've considered**
Manually changing the styling

**Comment by blvdmitry** [comment]
Ready for the next release for TextField and TextArea. I went with the Aligner approach to make sure DX is aligned with other components. To make sure there is not much additional overhead though, I've added a common internal aligner utility that works across mutliple component. Keeping it internal since there are dependencies on data-attributes and css variables to get the padding value instead of hardcoding them.

https://twitter.com/blvdmitry/status/1713653015742202316

![Image](https://github.com/formaat-design/reshaped/assets/887379/93e7481d-9918-47ef-9dd6-955278a9b365)
